File: scripts/query.js
const sqlite3 = require("sqlite3").verbose();
let db = new sqlite3.Database("./databases/trackrat.db", (err) => {
	if (err) {
		console.error(err.message);
	}
	console.log("Connected to the trackrat database.");
});

db.serialize(() => {
	db.each(
		`SELECT origin, current_location, destination FROM tracking WHERE tracking_num = 1`,
		(err, row) => {
			if (err) {
				console.error(err.message);
			}
			console.log(row);
		}
	);
});

db.close((err) => {
	if (err) {
		console.error(err.message);
	}
	console.log("Close the database connection.");
});

File: scripts/query.py
import sqlite3
from flask import Flask, render_template

app = Flask(__name__)

@app.route('/')
def index():
    # Connect to the database
    conn = sqlite3.connect('.databases/trackrat.db')
    c = conn.cursor()

    # Execute the query
    c.execute('SELECT * FROM tracking')

    # Fetch the results
    data = c.fetchall()

    # Close the connection
    conn.close()

    # Render the results in an HTML template
    return render_template('../pages/package.html', data=data)

if __name__ == '__main__':
    app.run()